src/structures/ReactionCollector.js

```diff
@@ -103,17 +103,15 @@ class ReactionCollector extends Collector {
    */
   dispose(reaction, user) {
     /**
-     * Emitted whenever a reaction is disposed of and the `dispose` option is set to true.
+     * Emitted when the entire reaction is removed and the `dispose` option is set to true.
```

Comment:
```suggestion
     * Emitted when the reaction had all the users removed and the `dispose` option is set to true.
```

Reply:
I tend to disagree, the wording should be changed but "removing a user" just doesn't feel right, I don't remove myself from the reaction, I remove my reaction, and a user removes their reaction, and when all reactions are removed at once it is very much like that.

Reply:
No. You remove yourself from users who reacted.
Message has reactions. Each reaction has an emoji. Each reaction has collection of users who reacted to it. If you un-react, you don't delete reaction at all. Reaction happens to be removed if there are 0 users who reacted with that emoji.